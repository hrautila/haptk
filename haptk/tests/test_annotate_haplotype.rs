mod common;

// use std::path::PathBuf;

// #[test]
// #[cfg(feature = "clap")]
// fn annotate_haplotype() {
//     use haptk::clap::LogAndVerbosity;

//     let cmd = haptk::clap::SubCommand::AnnotateHaplotype {
//         file: PathBuf::from(common::TEST_HAPLOTYPE),
//         ann: PathBuf::from("tests/data/test.gtf.gz"),
//         outdir: PathBuf::from("tests/results/annotated_haplotype.csv"),
//         log_and_verbosity: LogAndVerbosity {
//             verbosity: 1,
//             log_file: None,
//         },
//     };
//     haptk::clap::run_cmd(cmd).unwrap();

//     let res = std::fs::read_to_string("tests/results/annotated_haplotype.csv").unwrap();
//     insta::assert_yaml_snapshot!(res);
// }
