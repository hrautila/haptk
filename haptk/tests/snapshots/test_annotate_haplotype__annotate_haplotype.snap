---
source: haptk/tests/test_annotate_haplotype.rs
expression: res
---
"chr9\t18\tG\tT\t0\t\nchr9\t19\tA\tC\t0\t\nchr9\t20\tG\tT\t0\t\nchr9\t21\tA\tC\t0\t\nchr9\t22\tG\tT\t0\t\nchr9\t23\tA\tC\t0\t\nchr9\t24\tG\tT\t0\t\nchr9\t25\tA\tC\t0\t\nchr9\t26\tG\tT\t0\t\nchr9\t27\tA\tC\t0\t\nchr9\t28\tG\tT\t0\tcore\nchr9\t29\tA\tC\t0\tcore\nchr9\t30\tG\tT\t0\tcore\nchr9\t31\tA\tC\t0\tcore\nchr9\t32\tG\tT\t1\tcore\nchr9\t33\tA\tC\t0\tcore\nchr9\t34\tG\tT\t0\tcore\nchr9\t35\tA\tC\t0\tcore\nchr9\t36\tG\tT\t0\t\nchr9\t37\tA\tC\t0\t\nchr9\t38\tG\tT\t0\t\nchr9\t39\tA\tC\t0\t\nchr9\t40\tG\tT\t0\t\nchr9\t41\tA\tC\t0\t\nchr9\t42\tG\tT\t0\t\nchr9\t43\tA\tC\t0\t\nchr9\t44\tG\tT\t0\t\nchr9\t45\tA\tC\t0\t\nchr9\t46\tG\tT\t0\t\n"

